src/focus.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react'
+import css from 'styled-jsx/css'
+import { c_NEONS } from './theme'
+
+let count = 0
+
+const generateFocus = num => {
+  if (typeof num !== 'number') {
+    num = count++
+  }
+
+  return {
+    ...css.resolve`
+      button:focus,
+      *[href]:focus,
+      input:focus,
+      select:focus,
+      textarea:focus,
+      *[tabindex]:not([tabindex='-1']):focus {
+        position: relative;
+        display: inline-block;
+        outline: none;
+      }
+
+      button:focus::before,
+      *[href]:focus::before,
+      input:focus::before,
+      select:focus::before,
+      textarea:focus::before,
+      *[tabindex]:not([tabindex='-1']):focus::before {
+        border: 1px solid white;
+        outline: 1px solid ${c_NEONS[num % c_NEONS.length]};
```

Comment:
Trying it on my computer, I quite like it with the colour being 2px and the white 1px so the colour still dominates the eye